packages/button/stories/index.stories.mdx

```diff
@@ -142,6 +143,24 @@ Important notes:
 - `@click` on `<lion-button>` and `@submit` on `<form>` are triggered by these use cases. We strongly encourage you to listen to the submit handler if your goal is to do something on form-submit.
 - To prevent form submission full page reloads, add a **submit handler on the form** `@submit` with `event.preventDefault()`. Adding it on the `<lion-button>` is not enough.
 
+### Usage with lion inputs
```

Comment:
We usually try to avoid that our package depend on one another, and then for demos where we combine multiple components, we usually put them inside the form-system package, so we only have a single package that depends on all the others. I think the demo is nice, but then maybe we can add it inside `form-system` package as a separate commit?